### DISCUSSION TITLE
Two new icons - push pin and scatter plot

### DISCUSSION THREAD

**Discussion by dejanblagic** [tailwindlabs/heroicons#595] [Ideas]
1. Push pin icon that looks similar to this 📌 or this icon from Material
![image](https://user-images.githubusercontent.com/20821993/152848639-1bf441d3-c45f-441f-812a-7d0b08a932ba.png)

2. Dots (scatter plot) icon like this (Material)
![image](https://user-images.githubusercontent.com/20821993/152848767-8579c1f2-52af-49b4-848e-74c215814ee7.png)

**Comment by santyas** [comment]
Please add a pin or tack to symbolize an attached message ie. Thanks!
